config/routes.rb

```diff
@@ -145,9 +145,8 @@
   get 'search/dynamic' => 'searches#dynamic'
   get 'search/dynamic/:id' => 'searches#dynamic'
   get 'search/:id' => 'searches#results'
-  get 'search' => 'searches#new'
+  get 'search' => 'searches#new', as: 'search_query'
   post 'search' => 'searches#new'
```

Comment:
@jywarren I have done it in a get method. But still, there is this post method which I don't know if it is using or not.  I also have kept ` get 'search/:id' => 'searches#results'` although I dont think it will be use any more. What do you think I should do ?

Reply:
Hi, i think it's still a good idea to allow for `http://publiclab.org/search/sensors` for example. Does your current approach use `?q=sensors` instead?

Reply:
Hi yes `http://publiclab.org/search/sensors` this is still can be done. But now the search boxes search which `?q=sensors` as to be able to give character escaping abilities